CHANGELOG.md

```diff
@@ -6,6 +6,15 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.2.2]
+
+* Fixes write of start/stop sensing times due to changes in ASF Search v5.0.0 (see #79); schema requires 6 trailing digits
+
+
+## [0.2.1]
+
+* Fixes write of start/stop sensing times due to changes in ASF Search v5.0.0 (see #79)
+
```

Comment:
Yes, these should be combined. If there's more than one release past whatever is shown here:
https://github.com/ACCESS-Cloud-Based-InSAR/DockerizedTopsApp/releases/latest

The second PR should have just added the changes to the unreleased version. This is a common mistake, unfortunately, so I might have to figure out how to adjust the changelog actions to make it easier.